wordpress-mobile/WordPress-Android
# Issue/6507 third party authentication

#### Note
This should be merged ***after*** https://github.com/wordpress-mobile/WordPress-Android/pull/6522.

### Fix
Add third-party authentication screen as described in https://github.com/wordpress-mobile/WordPress-Android/issues/6507.  See the screenshots below for details.

![auth_screenshot_03_third_party](https://user-images.githubusercontent.com/3827611/29302431-40d86490-8141-11e7-8d5e-641201ef36f1.png)

### Test
0. Clear app data.
1. Tap ***Log In*** button.
2. Tap ***Log in with Google*** button.
3. Notice third-party authentication view.
4. Rotate device into landscape orientation.
5. Notice third-party authentication view.
6. Tap ***Help*** action.
7. Notice help screen is displayed.
